pkg/minishift/addon/manager/addon_manager_test.go

```diff
@@ -182,6 +185,109 @@ func TestVerifyMissingRequiredVariablesInContext(t *testing.T) {
 	}
 }
 
+type FakeSSHDockerCommander struct {
+	docker.DockerCommander
+	provision.SSHCommander
```

Comment:
@budhrg Doesn't need something like this?
```

	ocRunner             *oc.OcRunner
	dockerCommander      docker.DockerCommander
	sshCommander         provision.SSHCommander
	interpolationContext InterpolationContext
```

Reply:
See https://github.com/minishift/minishift/pull/1941#discussion_r164434817